kubernetes/template-generator/container_os.py

```diff
@@ -0,0 +1,58 @@
+class ContainerOS(object):
+    """
+        Baseclass defining OS for templating system
+    """
+
+    def is_systemd(self):
+        """
+            tells if the system is using systemd
+
+            @return Boolean     true, if OS is using systemd
+        """
+        raise NotImplemented()
+
+    def base_image(self):
+        """
+            returns the name of base docker image
+
+            @return string
+        """
+        raise NotImplemented()
+
+
+class UpstartContainerOS(ContainerOS):
+    """
+        Baseclass for upstart based OS
+    """
+    def is_systemd(self):
+        return False
+
+    def base_image(self):
+        return "gazdown/leapp-scratch"
```

Comment:
this seems a bit hard coded, i would consult with @vinzenz  and @mfranczy  how to change this, also if it absolutely has to stay hardcoded it should get under leapp, looks more professional.